### PR TITLE
Release 5.0.0

### PR DESCRIPTION
**Pull request by foarsitter**
Time for a release!

### CODE REVIEW

**Comment by gmcrocetti** [comment]
> Time for a release!

HUGE 🚀

**Comment by carljm** [comment]
I would like to be removed as a lead for this project, but I don't know how that's done. I haven't worked with Django or django-model-utils for quite a few years now, and wouldn't even feel competent to evaluate the state of this project (let alone having any idea how to make a release.)

**Comment by foarsitter** [comment]
Maybe @jezdez can be of help and remove you?

File: CHANGES.rst [hunk 1]
```diff
@@ -1,8 +1,11 @@
 Changelog
 =========
+..
+    To be released
+    --------------
```
Comment (charettes) [inline CHANGES.rst:5]: Looks like the indentation is broken?

**Review by charettes — COMMENTED** [review]
@foarsitter It looks like [there aren't any packages that were uploaded to Jazzband](https://jazzband.co/projects/django-model-utils) for me to review?

If you bump the date to tomorrow, address the indentation issue, merge this PR, and tag it appropriately as 5.0 I can approve the package uploaded to Jazzband.

**Comment by charettes** [comment]
Also, I'm all in for making you lead and replace Carl if he no longer wants to be involved. It's just not something I'm allowed to do myself.

File: CHANGES.rst (hunk 1 above)
Comment (foarsitter) [reply CHANGES.rst:5]: Commented the `To be released` heading because it looks sloppy when there is a new release and the heading is empty.

**Comment by foarsitter** [comment]
@charettes the release is available now on jazzband. Do you know who is allowed to make changes to the leads?

**Comment by charettes** [comment]
> the release is available now on jazzband

@foarsitter just [uploaded to Pypi](https://pypi.org/project/django-model-utils/#history)

> Do you know who is allowed to make changes to the leads?

It looks like it happened through @jezdez at the time https://github.com/jazzband/help/issues/151

**Pull request merged**